src/doc/rustc/src/command-line-arguments.md

```diff
@@ -86,6 +86,12 @@ This flag will set which lints should be set to the [deny level](lints/levels.ht
 
 This flag will set which lints should be set to the [forbid level](lints/levels.html#forbid).
 
+## `-Z`: set options for debugging rustc
```

Comment:
So, while it's true that many flags are good for debugging, that's not the purpose of `-Z`. It's really for *unstable* compiler flags. It just so happens that debugging is a good use of flags that aren't yet stable.

Could we maybe re-word this?

Reply:
Ahh glad you say so! I came to that conclusion by running `rustc --help` which says:

```
-Z help             Print internal options for debugging rustc
```

Maybe this should be changed as well then? What do you think about:

```
-Z help             Print unstable compiler flags
```

When running `rustc -Z help` it also starts with:
```
Available debug options:
...
```
Should this be changed into something more generic like "Available options:" as well?

Curious to what you think of all this! Because of these lines I was really under the impression it was for debug options only (even though that seemed a bit weird to me..)

Reply:
cc @rust-lang/compiler

Reply:
+1 -- these aren't just for debugging, changing the description would make sense to me.

Reply:
@steveklabnik @cramertj I updated the descriptions (Y).

I saw that some methods also contained words like "debug_..." but I wasn't brave enough to go and change those as well :)